.chloggen/ottl-use-stringlikegetter.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
Adding the `Concat` changes makes this a breaking change since the same inputs will produce a different output now.

Reply:
Updated